### PR TITLE
Admin UI: Updated to Admin UI version 1.24.4

### PR DESCRIPTION
**Pull request by amotl**
On Admin UI, this updates recent versions of engine.io, socket.io, and socket.io-parser. Nothing big.

### CODE REVIEW

File: docs/appendices/release-notes/unreleased.rst [hunk 1]
```diff
@@ -222,6 +222,9 @@ Administration and Operations
 - Added the ability to set a
   :ref:`storage_class <sql-create-repo-s3-storage_class>` for S3 repositories.
 
+- Updated to Admin UI version 1.24.4, updating to recent versions of engine.io,
```
Comment (seut) [inline docs/appendices/release-notes/unreleased.rst:225]: I think we shouldn't mention this as afaik this has no user-visible effect.

**Review by seut — APPROVED** [review]
As said, I suggest to remove the changes entry.

File: docs/appendices/release-notes/unreleased.rst (hunk 1 above)
Comment (matriv) [reply docs/appendices/release-notes/unreleased.rst:225]: +1

**Review by matriv — APPROVED** [review]

File: docs/appendices/release-notes/unreleased.rst (hunk 1 above)
Comment (amotl) [reply docs/appendices/release-notes/unreleased.rst]: I've removed the changelog item again. Thanks.

**Pull request merged**